assemblies/assembly-install-rhdh-ocp.adoc

```diff
@@ -5,19 +5,33 @@
 
 toc::[]
 
-You can install {product} on {ocp-short} by using one of the following methods:
+You can install {product} on {ocp-short} by using one of the following installers:
 
-* The Helm chart
 * The {product} Operator
+* The Helm chart
 
-// Helm chart method
-include::modules/installation/proc-install-rhdh-ocp-helm.adoc[leveloffset=+1]
-include::modules/installation/proc-installing-rhdh-on-openshift-with-helm-cli.adoc[leveloffset=+2]
-include::modules/getting-started/proc-add-custom-app-file-openshift-helm.adoc[leveloffset=+2]
-include::modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc[leveloffset=+2]
+== Installing {product} on {ocp-short} by using the operator
+
+You can install {product} on {ocp-short} by using the operator with one of the following methods:
 
-// Operator method
-include::modules/installation/proc-install-rhdh-ocp-operator.adoc[leveloffset=+1]
+* The {ocp-short} console
+* The {ocp-short} CLI
+
+include::modules/admin/proc-install-operator.adoc[leveloffset=+2]
+include::modules/installation/proc-install-rhdh-ocp-operator.adoc[leveloffset=+2]
 include::modules/installation/proc-add-custom-app-config-file-ocp-operator.adoc[leveloffset=+2]
 include::modules/installation/proc-config-dynamic-plugins-rhdh-operator.adoc[leveloffset=+2]
```

Comment:
This bit about configuring dynamic plugins with the Operator will not be part of the installation chapter once we move all installation content to the _Installing RHDH_ guide. There is an existing Jira to move that context to the Dynamic Plugins guide. https://issues.redhat.com/browse/RHIDP-2544

Reply:
Good. That's out of the scope of this PR.